build-logic/convention/src/main/kotlin/AndroidApplicationTestOptionsConventionPlugin.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.android.build.api.dsl.ApplicationExtension
+import com.google.samples.apps.nowinandroid.configureTestOptions
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.configure
+
+class AndroidApplicationTestOptionsConventionPlugin : Plugin<Project> {
```

Comment:
As we are configuring the unitTests, I would suggest to use the following name:
AndroidApplicationsTestOptionsUnitTestsConventionPlugin